website/docs/guides/bigquery-qs.md

```diff
@@ -86,28 +214,53 @@ In order to let dbt connect to your warehouse, you'll need to generate a keyfile
+    * **Username** &mdash; The username you created for Snowflake. The username is not your email address and is usually your first and last name together in one word. 
+    * **Password** &mdash; The password you set when creating your Snowflake account.
+    * **Schema** &mdash; You’ll notice that the schema name has been auto created for you. By convention, this is `dbt_<first-initial><last-name>`. This is the schema connected directly to your development environment, and it's where your models will be built when running dbt within the Cloud IDE.
+    * **Target name** &mdash; Leave as the default.
+    * **Threads** &mdash; Leave as 4. This is the number of simultaneous connects that dbt Cloud will make to build models concurrently.
+
+    <Lightbox src="/img/snowflake_tutorial/dbt_cloud_snowflake_development_credentials.png" title="dbt Cloud - Snowflake Development Credentials" />
+
+6. Click **Test Connection**. This verifies that dbt Cloud can access your Snowflake account.
+7. If the connection test succeeds, click **Next**. If it fails, you may need to check your Snowflake settings and credentials.
+
+</TabItem>
+</Tabs>
 
 ## Set up a dbt Cloud managed repository 
-<Snippet path="tutorial-managed-repo" />
+If you used Partner Connect, you can skip to [initializing your dbt project](#initialize-your-dbt-project-and-start-developing) as the Partner Connect provides you with a managed repository. Otherwise, you will need to create your repository connection. 
 
+<Snippet path="tutorial-managed-repo" />
 
 ## Initialize your dbt project​ and start developing
 Now that you have a repository configured, you can initialize your project and start development in dbt Cloud:
 
 1. Click **Start developing in the IDE**. It might take a few minutes for your project to spin up for the first time as it establishes your git connection, clones your repo, and tests the connection to the warehouse.
-2. Above the file tree to the left, click **Initialize dbt project**. This builds out your folder structure with example models.
-3. Make your initial commit by clicking **Commit and sync**. Use the commit message `initial commit` and click **Commit**. This creates the first commit to your managed repo and allows you to open a branch where you can add new dbt code.
+2. Above the file tree to the left, click **Initialize your project**. This builds out your folder structure with example models.
+3. Make your initial commit by clicking **Commit and sync**. Use the commit message `initial commit`. This creates the first commit to your managed repo and allows you to open a branch where you can add new dbt code.
 4. You can now directly query data from your warehouse and execute `dbt run`. You can try this out now:
-    - Click **+ Create new file**, add this query to the new file, and click **Save as** to save the new file:  
+    - Click **+ Create new file**, add this query to the new file, and click **Save as** to save the new file: 
```

Comment:
Looks like these UI elements are bolded even tho it's flagged as not

Reply:
i think i'm going to remove the UI elements config bc it's not really working well. I'll try to get it working in the next iteration